### ISSUE TITLE
Use "load_from" property to train my custom model from another model I trained before. 使用"load_from"参数在我已经训练过的模型上继续训练

### ISSUE THREAD

**Issue by xiangwentao666** [open-mmlab/mmaction2#1583]
Thank you for this toolbox! But I got a problem when trying **to use "load_from" property to train my custom model from another model trained before.**
感谢您的这个工具箱！但是我在尝试配置 load_from 这个属性去继续训练某个我已经训练过的模型的时候，遇到了一个问题。

As for the config file, the only property I changed is the property "load_from", the value of which is changed from "None" to the path to my pre-trained model. When the first epoch finished, the value of "loss" shown in the log history is 3.7337, while the value of "loss" of the pre-trained model shown in the log history is 0.5434. Does anyone know why it is much larger than the loss value of the pre-trained model?
在config文件中，我只修改了 load_from 这一个属性，原本是None、我改成了我训练过的那个模型文件的路径。但是第一个epoch结束后显示的loss值与训练过的那个模型的最后一个epoch显示的loss值不一样，第一个epoch的值是3.7337、之前训练过的那个模型的最后一个epoch显示的loss值是0.5434。请问有人知道为什么使用load_from加载的模型的损失值为何与之前训练过的模型的loss值不一样吗？

**Comment by kennymckormick** [comment]
Hi, Xiangwentao, 
Using load_from, you just initialize the model with a given checkpoint file and start training from the beginning. There's no guarantee that you will have a smaller loss. (For example, you use a large lr  at the beginning and the model soon deviate from the good local minimum).

**Comment by xiangwentao666** [comment]
> Hi, Xiangwentao, Using load_from, you just initialize the model with a given checkpoint file and start training from the beginning. There's no guarantee that you will have a smaller loss. (For example, you use a large lr at the beginning and the model soon deviate from the good local minimum).

Well, thank you for your timely reply!
嗯嗯好，感谢您及时的回复！
So, I guess, that a large difference may exist between a model trained for 300 epochs in a whole run and a model trained 3 times with all property unchanged except for load_from and total_epochs ( 100 epochs for each time, using load_from property ). Anything wrong?
我猜的话，通过这个方式得到的两个模型是不一样的：一个是一次性连续训练300个epoch、另一个是只改变load_from和total_epochs这两个参数不改变其他所有属性的情况下运行3次并且每次运行均训练100个epoch。是这样的吗？

**Comment by kennymckormick** [comment]
> > Hi, Xiangwentao, Using load_from, you just initialize the model with a given checkpoint file and start training from the beginning. There's no guarantee that you will have a smaller loss. (For example, you use a large lr at the beginning and the model soon deviate from the good local minimum).
> 
> Well, thank you for your timely reply! 嗯嗯好，感谢您及时的回复！ So, I guess, that a large difference may exist between a model trained for 300 epochs in a whole run and a model trained 3 times with all property unchanged except for load_from and total_epochs ( 100 epochs for each time, using load_from property ). Anything wrong? 我猜的话，通过这个方式得到的两个模型是不一样的：一个是一次性连续训练300个epoch、另一个是只改变load_from和total_epochs这两个参数不改变其他所有属性的情况下运行3次并且每次运行均训练100个epoch。是这样的吗？

Yes, they are different. You can try to figure out the difference between load_from and resume_from

**Comment by xiangwentao666** [comment]
> > > Hi, Xiangwentao, Using load_from, you just initialize the model with a given checkpoint file and start training from the beginning. There's no guarantee that you will have a smaller loss. (For example, you use a large lr at the beginning and the model soon deviate from the good local minimum).
> > 
> > 
> > Well, thank you for your timely reply! 嗯嗯好，感谢您及时的回复！ So, I guess, that a large difference may exist between a model trained for 300 epochs in a whole run and a model trained 3 times with all property unchanged except for load_from and total_epochs ( 100 epochs for each time, using load_from property ). Anything wrong? 我猜的话，通过这个方式得到的两个模型是不一样的：一个是一次性连续训练300个epoch、另一个是只改变load_from和total_epochs这两个参数不改变其他所有属性的情况下运行3次并且每次运行均训练100个epoch。是这样的吗？
> 
> Yes, they are different. You can try to figure out the difference between load_from and resume_from
Got it! Thanks for your timely help!
